### PR TITLE
Add Halon production to electrolyzer reactions, remove normal one

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Remove halon reaction from gas reactions.
Added it to the electrolyzer reactions slightly modified to become a tier 1 gas, now is made with co2 and n2o at low temperature and low pressure (similar to BZ)

## Why It's Good For The Game
Allow earlier and easier uses of this gas that is supposed to be helping stop fires

## Changelog

:cl:
remove: removed halon from normal gas reactions
expansion: added halon to the electrolyzer reactions by consuming co2 and n2o at low temperature and low pressure
/:cl:

### CODE REVIEW

File: code/modules/atmospherics/machinery/components/electrolyzer/electrolyzer_reactions.dm [hunk 1]
```diff
@@ -60,4 +60,27 @@ GLOBAL_LIST_INIT(electrolyzer_reactions, electrolyzer_reactions_list())
 	air_mixture.assert_gases(/datum/gas/hypernoblium, /datum/gas/antinoblium)
 	var/proportion = min(air_mixture.gases[/datum/gas/hypernoblium][MOLES], (1.5 * (working_power ** 2)))
 	air_mixture.gases[/datum/gas/hypernoblium][MOLES] -= proportion
-	air_mixture.gases[/datum/gas/antinoblium][MOLES] += proportion * 0.5
+	air_mixture.gases[/datum/gas/antinoblium][MOLES] += proportion * 0.05
```
Comment (JohnFulpWillard) [inline code/modules/atmospherics/machinery/components/electrolyzer/electrolyzer_reactions.dm:63]: was this intended?

Comment (Ghilker) [reply code/modules/atmospherics/machinery/components/electrolyzer/electrolyzer_reactions.dm:63]: Ops was thinking of changing this one to lower it a bit (is antinob, should not be this easy, and I was supposed to put it at 0.01) will revert it

Comment (Ghilker) [inline code/modules/atmospherics/machinery/components/electrolyzer/electrolyzer_reactions.dm:63]:
```suggestion
	air_mixture.gases[/datum/gas/antinoblium][MOLES] += proportion * 0.5
```

**Comment by Cheshify** [comment]
Incredibly based and good PR Ghilker, this is perfectly in line with what I think atmospherics needs

**Comment by Kylerace** [comment]
how is this an expansion

**Comment by Pickle-Coding** [comment]
The heat capacity in this reaction will increase, putting a limit on where the temperature will converge. The limit is almost below TCMB due to the low energy release from the reaction. The reaction is more efficient at lower pressures (decreasing temperature will increase reaction efficiency).

To keep it interesting, I suggest increasing the energy release so it doesn't converge to such low temperatures, and instead fight against its optimal production rate.

**Comment by Ghilker** [comment]
ah fuck

**Comment by Ghilker** [comment]
i uh sorry people

**Comment by rdragan** [comment]
try invoking a sink

**Comment by Ghilker** [comment]
I think I'll close this one and remake it cause this is messed up badly

**Comment by LemonInTheDark** [comment]
chad

**Pull request closed without merging**